lib/src/platform_client.dart

```diff
@@ -917,12 +917,17 @@ class PlatformClient {
 
   Future<Map<String, String>> _getHeaders(int traceId, {Map<String, String>? additionalHeaders}) async {
     final headers = {
-      'Content-Type': 'application/json',
+      HttpHeaders.contentTypeHeader: 'application/json',
       'X-API-Key': _config.apiKey,
       'X-Client-Id': _config.clientId,
       'X-Trace-Id': traceId.toString(),
     };
 
+    if (!kIsWeb) {
+      // The http package does not automatically set the Accept-Language header on mobile.
+      headers[HttpHeaders.acceptLanguageHeader] = Platform.localeName.replaceAll('_', '-');
```

Comment:
`Platform.localeName` returns a string like `en_US`, but the [`Accept-Language`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Language) header expects a `-`.